src/interface/src/app/home/home.component.html

```diff
@@ -1,2 +1,4 @@
-<app-region-selection></app-region-selection>
-<app-plan-table></app-plan-table>
+<app-region-selection
+  *ngIf="(loggedIn$ | async) === false"
+></app-region-selection>
+<app-plan-table *ngIf="loggedIn$ | async"></app-plan-table>
```

Comment:
I think the home/welcome text lives in this table (toggle between plan list and welcome text is based off of the login flag). If we're only showing the table for logged in users, has the welcome text moved somewhere else for not logged in users?

Reply:
This part is the `<app-region-selection>`
<img width="1016" alt="Screen Shot 2023-09-01 at 14 23 35" src="https://github.com/OurPlanscape/Planscape/assets/358892/f66b195d-0857-4bd5-b392-9dcf7b8338ac">

Reply:
Yeah, sorry if I misplaced the comment, I was talking about line 4: <app-plan-table *ngIf="loggedIn$ | async"></app-plan-table>

Reply:
right but not sure i follow.. we have 2 "welcome to planscape preview" texts now. The one on region selection and the one above.
Do we need both? Is that the piece you mention is missing?

Reply:
talked this out on a call and figured out what I was missing, thanks @sachiyuan will mark this as draft until is good to go